### PR TITLE
feat: add filled check circle icon and larger icon size

### PR DESCRIPTION
**Pull request by jake-bassett**
## Description
New icon for a filled checked circle and a larger icon size.

### CODE REVIEW

File: projects/components/src/icon/icon-size.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ export const enum IconSize {
   Medium = 'medium',
   Large = 'large',
   ExtraLarge = 'extra-large',
+  ExtraLarger = 'extra-larger',
```
Comment (anandtiwary) [inline projects/components/src/icon/icon-size.ts:7]: We need better names

**Review by anandtiwary — APPROVED** [review]

**Comment by anandtiwary** [comment]
Should we switch to L, XL, XXL etc ?

File: projects/components/src/icon/icon-size.ts (hunk 1 above)
Comment (jake-bassett) [reply projects/components/src/icon/icon-size.ts:7]: Oh, and here I was hoping to squeeze in an `ExtraMoreLargerest` at some point.

**Comment by jake-bassett** [comment]
> Should we switch to L, XL, XXL etc ?

Yeah, if we add any more we should move to something else. It's not too bad yet. Dare I say, even a little fun. Any more though and it will be too much.

**Pull request merged**